pkg/sql/logictest/testdata/logic_test/fk

```diff
@@ -4119,3 +4119,39 @@ DROP TABLE drop_fk_during_addition_ref;
 
 statement error pgcode XXA00 pq: transaction committed but schema change aborted with error: \(42P01\): referenced relation \"drop_fk_during_addition_ref\" does not exist
 COMMIT;
+
+# Regression test for #80828. Do not attempt a fast path cascading delete when
+# a descendent is in the process of being added.
+subtest 80828
+
+statement ok
+CREATE TABLE parent80828 (id int primary key);
+CREATE TABLE child80828 (id INT PRIMARY KEY, pid INT NOT NULL REFERENCES parent80828(id) ON DELETE CASCADE, UNIQUE INDEX (pid));
+INSERT INTO parent80828 VALUES (1);
+INSERT INTO child80828 VALUES (1, 1);
+
+statement ok
+SET CLUSTER SETTING jobs.registry.interval.base = .00001;
```

Comment:
you probably want to reset this one too.

Reply:
Done.